projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -271,6 +271,13 @@ deploymentDockerRegistryUsernameReadOnly: ""
 deploymentDockerRegistryPasswordReadOnly: ""
 
 
+## [Required if deploymentDockerRegistryType=jfrog]
+## Connection settings for the Jfrog artifactory instance, used to check if a data job image already exists
+jfrogArtifactoryUrl: ""
```

Comment:
It is not clear if these properties will be used exclusively for deployment. Alternatively, do we need separate properties for each registry type, or can we reuse the one for generic?

Reply:
Comment extended for clarification.